roles/oodmatlab/tasks/main.yaml

```diff
@@ -0,0 +1,15 @@
+- name: matlab directory
```

Comment:
In changing the name of the role form oodmatlab, did you use git mv or just move files and add new ones?  It's better to rename files with git so that the history of edits are maintained.

As such, it appears that you still have two distinct role names in your repo, "ood_matlab" and "oodmatlab".